scopes/workspace/workspace-config-files/ws-config.cmd.ts

```diff
@@ -124,9 +124,9 @@ export class WsConfigCleanCmd implements Command {
     [
       'w',
       'writers <writers>',
-      `only write config files for the given writers. use comma to separate multiple writers. use ${COMMAND_NAME} list to see all writers`,
+      `only clean config files for the given writers. use comma to separate multiple writers. use ${COMMAND_NAME} list to see all writers`,
```

Comment:
Why clean? (the previous flag is clean, but this one is write)

Reply:
actually this is for the clean command, so guess this should be clean (and was a copy-paste typo)